src/Meta/MySql/Schema.php

```diff
@@ -265,19 +265,12 @@ protected function resolveForeignTable($table, Blueprint $blueprint)
 
     /**
      * @param \Illuminate\Database\Connection $connection
-     *
+     * @deprecated use \Reliese\Meta\MySql\Database::getSchemaNames instead
      * @return array
      */
     public static function schemas(Connection $connection)
     {
-        $schemas = $connection->getDoctrineSchemaManager()->listDatabases();
-
-        return array_diff($schemas, [
-            'information_schema',
-            'sys',
-            'mysql',
-            'performance_schema',
-        ]);
+        return (new Database($connection))->getSchemaNames();
```

Comment:
This method's cohesion is very low compared to the rest of the methods. It makes sense to put this logic elsewhere.